src/main/run_work.py

```diff
@@ -54,40 +75,65 @@ def _draft_config(path):
         if user_in.lower() in "RrFf":
             break
     if user_in.lower() == "f":
+        return True
+    else:
+        return False
+
+def _display_draft(draft):
+    outputdraft = draft["outputDraft"]
+    filedraft = draft["fileDraft"]
+    display_status_symbol(2,2,f"fileDraft: {filedraft}")
+    display_status_symbol(2,2,f"outputDraft: {outputdraft}",True)
+
+def _get_draft(path,draft_config):
+    if draft_config:
         draft = CallApi(path).fetch()
-        print(draft)
+        display_status_symbol(1,2,"Choosen draft")
+        _display_draft(draft)
     else:
         with open(os.path.join(path, "ta", "draft.json"), "r") as draftfile:
             draft = json.load(draftfile)
             draftfile.close()
+        display_status_symbol(1,2,"Choosen draft")
+        _display_draft(draft)
+        
     return draft
 
 
-def _add_data_to_work(path, draft):
+def _add_data_to_work(path, draft, workId):
     work = Work()
     work.draft = draft
     work.path = path
-    work.workId = ConfigEditor(path=path).readconfig()["workId"]
+    work.workId = workId
     if work.property_is_ready():
         work_path = os.path.join(path, "ta", "work.json")
         if work.create():
-            print(f" |-[/] {work_path} created")
+            display_status_symbol(1, 0, f"{work_path} created")
         else:
-            print(f" |-[X] {work_path} already exists")
+            display_status_symbol(1, 2, f"{work_path} already exists")
     else:
-        print("property is not ready")
-        print(work.draft)
-        print(work.path) 
-        print(work.workId)
+        display_status_symbol(1, 1, f"Error: Invalid component")
+        component = {"draft":work.draft,"path":work.path,"workId":work.workId}
+        count = 0
+        end = False
+        for k,i in component.items():
+            count += 1
+            if count == len(component):
+                end = True
+            if i == None:
+                display_status_symbol(2, 1, f"{k}",end)
+            else:
+                display_status_symbol(2, 0, f"{k}",end)
```

Comment:
Use the fill name or meaningful variable name make it easier to understand. Also simplify these one. just like the previous review.